tests/bearlib/aspects/InstanceTest.py

```diff
@@ -13,14 +13,14 @@ def test_tastes(
 
     def test__setattr__(self, SubAspect, SubAspect_tastes):
         aspect = SubAspect('py')
+        error_taste = "A 'taste' value for this aspectclass instance "\
+            'exists already.'
+        error_aspectless = "can't set attributes of aspectclass instances"
         for name in SubAspect_tastes:
             with pytest.raises(AttributeError) as exc:
                 setattr(aspect, name, 'value')
-            assert str(exc.value) \
-                == "A 'taste' value for this aspectclass instance "\
-                   'exists already.'
+            assert exc.match(error_taste)
```

Comment:
This change doesnt change anything. Actually it is less precise that the previous test code.

Reply:
(It does at least get rid of the trailing `\`, so I'll accept it ;-) )